main.c

```diff
@@ -3646,7 +3646,7 @@ ovl_setattr (fuse_req_t req, fuse_ino_t ino, struct stat *attr, int to_set, stru
       if (fd >= 0)
         ret = futimens (fd, times);
       else
-        ret = utimensat (AT_FDCWD, path, times, AT_SYMLINK_NOFOLLOW);
+        ret = utimensat (AT_FDCWD, path, times, 0);
```

Comment:
So this will set the Link files time stamp Not the link it was pointing to?

Reply:
we are using it to update the `/proc/PID/FD` entry which is a symlink but handled specially by the kernel, so we want to modify the symlink itself in this case.

It is very easy to get this wrong, so I've added some tests to avoid regressions in future.